semgrep-core/src/core/Pcre_settings.mli

```diff
@@ -20,6 +20,99 @@ val regexp :
   string ->
   Pcre.regexp
 
+(*
```

Comment:
What about renaming this file Pcre_noexn.ml?
Or Pcre_result?

Reply:
I don't like the name `Pcre_settings` for this either, it's misleading. Maybe `R2C_pcre`? `R2C` is the namespace we wish we had if OCaml had namespaces.

Reply:
nah, not a huge fan of adding R2C here. Maybe PPcre? I saw some people doing LList, MMap for wrappers.

Reply:
I'd much rather have a common prefix because it allows grouping, sorting, and grepping. One letter is fine, but should be always the same letter. The [containers library uses the two letters `CC`](https://github.com/c-cube/ocaml-containers/tree/master/src/core). I think it works well. It's unambiguous, doesn't shadow the original definitions, and it's not too long.

Maybe just `S` for `Semgrep`, giving us `SPcre`.

Reply:
Ok, I'm fine with SPcre.
cc @IagoAbal @emjin @mmcqd